### DISCUSSION TITLE
Umbraco + Next.js Starter Template

### DISCUSSION THREAD

**Discussion by colnick** [umbraco/Umbraco-CMS#16127] [Features and ideas]
With the addition of the Umbraco Delivery API, it would be great to see Umbraco implement a Starter Template using Next.js or similar when creating a new project.

**Comment by nul800sebastiaan** [comment]
FYI
- We have a PR open for Next.js https://github.com/vercel/next.js/pull/52777
- And a community PR for Astro was just created as well: https://github.com/withastro/docs/pull/7879
